### ISSUE TITLE
Website Preview - Card Embed

### ISSUE THREAD

**Issue by polc1410** [christopherkenny/bskyr#17]
Do you have any plans to implement: Website Card Embeding as described here:  https://docs.bsky.app/blog/create-post#website-card-embeds

If you don't have direct plans, would you be interested in possibly receiving a pull request (I haven't looked at your raw code yet to see how easy this will be!) perhaps in spring 2025 (I have a few projects on currently).

**Comment by christopherkenny** [comment]
Yes, actively working on this! I wasn't familiar with Open Graph until yesterday and it seems simple to add.

**Comment by christopherkenny** [comment]
This should be fully functional now! If there are ideas to improve it, PRs are always welcome.

**Comment by polc1410** [comment]
Cool.  I will get this on and tested.
Many thanks for doing this.
